.circleci/src/commands/notify.yml

```diff
@@ -31,7 +31,10 @@ steps:
   - run:
       command: |
         echo 'export ALLOW_ALL_BRANCHES="<< parameters.allow-all-branches >>"' >> $BASH_ENV
-        app-toolbelt v0 notify mattermost --message "${MM_MESSAGE}" --channel << parameters.channel >> ${ALLOW_ALL_BRANCHES:+--allow-all-branches}
-      name: Mattermost Status Alert
+      name: Set ALLOW_ALL_BRANCHES flag
       when: always
-
+  - run:
+      command: |
+        app-toolbelt v0 notify mattermost --message "<< parameters.success_message >>" --channel << parameters.channel >> <<#parameters.allow-all-branches>>--allow-all-branches<</parameters.allow-all-branches>>
```

Comment:
is this "#" correct here? `#parameters.allow-all-branches`

Reply:
yes it is correct, this is some circleci magic syntax to add the content between the both `<< >>` if the parameter is true.

Reply:
yes i know. but i'm talking about the additional `#` what does that mean?